content/docs/command-reference/plots/index.md

```diff
@@ -158,6 +158,23 @@ header (first row) are equivalent to field names.
 
 - `<DVC_METRIC_Y_TITLE>` (optional) - field name to display as the X axis label
 
+## HTML templates
+
+By default, the plots generated by `dvc plots` require Internet access to load
+Vega JavaScript libraries. It's possible to supply a custom HTML file to the
+`--html-template` option of `dvc plot show` and `dvc plot diff` when the
+Internet access is not available.
+
+> ⚠️ This is a separate feature from
+> [custom Vega templates](/doc/command-reference/plots#custom-templates).
+> `--html-template` provides a custom HTML file to inject plots generated by
+> DVC.
+
+You can create an HTML file with `<script>` references to local Vega libraries.
+Use `{plot_divs}` as a placeholder for DVC to inject the plot images. See
```

Comment:
> inject the plot images

Actually is it images exactly what gets injected? Maybe I got this wrong/inexact.

Reply:
Technically they are not the images, plots are produced by Vega library and probably they are drawn on HTML canvas, take user input for modification etc.